packages/react-focus/src/components/FocusZone/FocusZone.tsx

```diff
@@ -217,11 +217,10 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
 
   public render(): React.ReactNode {
     // tslint:disable-next-line:deprecation
-    const { rootProps, ariaDescribedBy, ariaLabelledBy, className } = this.props;
+    const { as: tag, elementType, rootProps, ariaDescribedBy, ariaLabelledBy, className } = this.props;
     const divProps = getNativeProps(this.props, htmlElementProperties);
 
-    // tslint:disable-next-line:deprecation
-    const Tag = this.props.as || this.props.elementType || 'div';
+    const Tag = tag || elementType || 'div';
```

Comment:
With composition coming down the pipes, should we hold off on this part and have it get handled by slots?

Reply:
@dzearing  and @levithomason to chime in

Reply:
@joschect This is doing the same thing it was doing before, I'm just using the aliases in line 220 instead to keep the `tslint:disable-next-line:deprecation` comments to a minimum.